### PR TITLE
feat: add --insecure-skip-tls-verify flag to the nexd cli tool

### PR DESCRIPTION
**Pull request by chirino**
add --insecure-skip-tls-verify flag to the nexd cli tool.  This can be handy if your testing against a nexodus server that is using self signed certs and your device has not installed those certs into the system certs.

chore: Uppercase first letter of all flag Usage to be consistent with how other tools like git and kubectl show help.

### CODE REVIEW

File: cmd/apiserver/main.go [hunk 1]
```diff
@@ -53,31 +53,32 @@ func init() {
 
 // @BasePath  		/api
 func main() {
+	cli.HelpFlag.(*cli.BoolFlag).Usage = "Show help"
```
Comment (russellb) [inline cmd/apiserver/main.go:56]: It took me a bit to figure out why this was needed. It's to get `Show` to be capitalized, right? I appreciate your dedication to getting the output consistent!

A comment would be helpful for the future so others know why this was needed.

**Review by russellb — APPROVED** [review]
One trivial comment, which I can just push a small commit to address

File: cmd/apiserver/main.go (hunk 1 above)
Comment (russellb) [reply cmd/apiserver/main.go:57]: pushed a commit to the branch for this

**Comment by chirino** [comment]
Yep.  It’s to upper case the help too

**Pull request merged**